src/Factories/OpenexchangeratesFactory.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace Lab19\Cart\Factories;
+
+use Lab19\Cart\Services\Openexchangerates;
+
+class OpenexchangeratesFactory
+{
+    public static function create($currency, $base)
+    {
+        $currencyObject = new Openexchangerates($currency, $base);
+        $currencyObject->setCurrency($currency);
+        $currencyObject->setBaseCurrency($base);
+        $currencyObject->setResponseFromOpenExhange(); //This function does the api call
```

Comment:
@AlexeiaW just looks like a spelling issue on this method `setResponseFromOpenExhange` should be `setResponseFromOpenExchange`

Reply:
I corrected the spelling